_docs_use-cases/use-case-transport-create-own-relationshiptemplate.md

```diff
@@ -49,7 +49,7 @@ A [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relat
 ## Parameters
 
 - `expiresAt` is the ISODateTime the RelationshipTemplate expires at. This should be set as small as possible and cannot be in the past.
-- `content` describes the structure of the RelationshipTemplate. Whereas the enmeshed App requires a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) as structure, the content of the RelationshipTemplate is arbitrary data.
+- `content` describes the structure of the RelationshipTemplate. The enmeshed App requires a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) as structure, otherwise an [ArbitraryRelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshiptemplatecontent) containing any data can be used.
```

Comment:
Please be more precise about the fact that if the Identity **receiving** the RelationshipTemplate is an App user, the creator of the RelationshipTemplate must use a RelationshipTemplateContent.